.drone.yml

```diff
@@ -47,5 +46,35 @@ steps:
         include:
           - master
       event:
+        include:
+          - push
+          - tag
         exclude:
           - pull_request
+
+  - name: gen-manifests
+    image: golang:latest
+    commands:
+      - wget -O /usr/local/bin/kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v1.0.11/kustomize_1.0.11_linux_amd64
+      - chmod +x /usr/local/bin/kustomize
+      - make manifests
+      - cat provider-components.yaml
+    environment:
+      IMG: "autonomy/cluster-api-provider-talos:${DRONE_TAG}"
+    when:
+      event:
+          include:
+            - tag
```

Comment:
Is it worth running this on every PR to make sure that manifest generation doesn’t get broken by a PR?

Reply:
yeah, makes sense. I'll move it up the chain and run it every time.

Reply:
err, actually, take that back. kustomize barfs if we run this when it's not a tag. because of the way we have to pass in the full image string. if its not a tag, the IMG env var is `autonomy/cluster-api-provider-talos:` and it has beef with the colon at the end.

Reply:
See https://github.com/talos-systems/cluster-api-provider-talos/pull/21#discussion_r289521059. I think that will fix this.